loky/backend/popen_loky_win32.py

```diff
@@ -109,6 +109,12 @@ def __init__(self, process_obj):
         except IOError as exc:
             # IOError 22 happens when the launched subprocess terminated before
             # wfd.close is called. Thus we can safely ignore it.
+            import traceback
+
+            tb = traceback.format_exception(
+                type(exc), exc, getattr(exc, "__traceback__", None)
+            )
+            util.debug("error in Popen.__init__:\n" + "".join(tb))
             if exc.errno != 22:
```

Comment:
Note that in cpython `main`, this special casing of `errno 22` does not exist: the exception is always raised.